demisto_sdk/commands/common/hook_validations/lists.py

```diff
@@ -0,0 +1,54 @@
+from distutils.version import LooseVersion
+from typing import List
+
+from demisto_sdk.commands.common.errors import Errors
+from demisto_sdk.commands.common.hook_validations.content_entity_validator import \
+    ContentEntityValidator
+
+FROM_VERSION_LISTS = '6.5.0'
+
+
+class ListsValidator(ContentEntityValidator):
+    def __init__(self, structure_validator, ignored_errors=False, print_as_warnings=False,
+                 json_file_path=None, **kwargs):
+        super().__init__(structure_validator, ignored_errors, print_as_warnings,
+                         json_file_path=json_file_path, **kwargs)
+        self.from_version = self.current_file.get('fromVersion')
+        self.to_version = self.current_file.get('toVersion')
+
+    def is_valid_list(self) -> bool:
+        """Check whether the list is valid or not.
+
+        Returns:
+            bool. Whether the list is valid or not
+        """
+        # Lists files have fromServerVersion instead of fromVersion
+        validations: List = [
+            self.is_valid_version(),
+            self.is_valid_from_server_version(),
+        ]
+
+        return all(validations)
+
+    def is_valid_version(self) -> bool:
```

Comment:
just call to _is_valid_version from is_valid_list, this function that calls a function isn't really needed